orchestra/google_apps/service.py

```diff
@@ -129,3 +129,47 @@ def delete_folder(self, folder_id):
         except errors.HttpError as error:
             logger.exception('An error occured: %s', error)
             return None
+
+    def list_folder(self, folder_id):
```

Comment:
we should add tests for these.  OK i you want to put that in a post-launch cleanup issue :)